seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/sink/DruidOutputFormat.java

```diff
@@ -64,25 +65,29 @@
     private static final String DEFAULT_TIMESTAMP_COLUMN = "timestamp";
     private static final String DEFAULT_TIMESTAMP_FORMAT = "auto";
     private static final DateTime DEFAULT_TIMESTAMP_MISSING_VALUE = null;
+    private static final long DEFAULT_BATCH_SIZE = 1024;
 
-    private final transient StringBuffer data;
+    private final StringBuffer data = new StringBuffer();
```

Comment:
In my knowledge, the `RichOutputFormat` in Flink will create for each thread, it's threadsafe, so we can just use StringBuilder and long here.

Reply:
In the [benchmark result](https://github.com/asdf2014/yuzhouwan/blob/master/yuzhouwan-hacker/src/test/java/com/yuzhouwan/hacker/algorithms/thread/sync/StringBufferTest.java#L45), it seems the performance of StringBuffer and StringBuilder have not obvious different.